src/tools/apollo-tool/apollo-tuner.cpp

```diff
@@ -221,7 +243,7 @@ extern "C" void kokkosp_request_values(
   // "<<numTuningVariables<<" tuning\n";
   int choiceSpaceSize = 1;
```

Comment:
It seems feasible to me that the space size could exceed a billion and overflow the int, but highly unlikely to exeed 64 bits. I would use a 64-bit here just to track overflow.

Reply:
Limitation of Apollo

Reply:
Yeah, more a question of if you overflow `max_choices`. If you made it a 64-bit int temporarily - even if it's not allowed to exceed 32-bits - you would be more robust detecting overflow.  Unlikely to occur, obviously.